db/migration/V002__Initial_Data.sql

```diff
@@ -1,9 +1,3 @@
 -- Create test participant
 INSERT INTO studyparticipant (subject_id) VALUES('7bfc3b07-a97d-4e11-8ac6-b970c1745476');
 INSERT INTO studyparticipant (subject_id) VALUES('dd06747c-03df-4adb-9dbb-90b38dccab16');
--- Create test questionnaires
---Please paste your questionniare inside the ''
```

Comment:
When you remove this part, you need to adjust the database README. There is a link which points on this in README under DB Scripts.